CHANGELOG

```diff
@@ -0,0 +1,22 @@
+## What's Changed
+* Increase support for unions in incremental smt2 decision procedure by @thomasspriggs in https://github.com/diffblue/cbmc/pull/7926
+* Add ASV book CBMC examples by @peterschrammel in https://github.com/diffblue/cbmc/pull/7921
+* Add models for C library: getpwnam, getpwuid, and getopt by @tautschnig in https://github.com/diffblue/cbmc/pull/7919 and https://github.com/diffblue/cbmc/pull/7916
+* Shadow memory addresses now also support --pointer-check argument by @esteffin in https://github.com/diffblue/cbmc/pull/7936
+* [DOCS] Add documentation on CBMC Shadow Memory intrinsics by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7913
+* [DOCS] Improve doxygen for shadow_memory_util by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7930
+* [SYNTHESIZER] Allow goto-synthesizer accept all CBMC options  by @qinheping in https://github.com/diffblue/cbmc/pull/7900
```

Comment:
I'd reword this as "goto-synthesizer now accepts all CBMC options by @qinheping ..."